### PR TITLE
Support for ssl

### PR DESCRIPTION
**Pull request by SonicDMG**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -21,3 +21,4 @@ config/production.yaml
 
 # Webstorm/IDEA
 .idea/
+.vscode
```
Comment (jeffreyscarpenter) [inline .gitignore:24]: Yeah baby - VS Code FTW!

File: Dockerfile [hunk 2]
```diff
@@ -16,7 +16,7 @@ WORKDIR /opt/killrvideo-generator
 # Copy package.json for dependencies
 COPY package.json /opt/killrvideo-generator/
 COPY npm-shrinkwrap.json /opt/killrvideo-generator/
-    
+
```
Comment (jeffreyscarpenter) [inline Dockerfile:19]: Did this file actually change (beyond whitespace?)

File: src/utils/cassandra.js [hunk 3]
```diff
@@ -66,22 +66,42 @@ export function getCassandraClientAsync(keyspace, dseUsername, dsePassword) {
...
 
         // Use the values passed in from the config
         clientOpts.authProvider = new auth.DsePlainTextAuthProvider(dseUsername, dsePassword);
 
       } else {
         logger.info('No detected username/password combination was passed in. DSE cluster authentication method was NOT executed.');
       }
-      
+
+      let Filesystem = require("fs");
```
Comment (jeffreyscarpenter) [inline src/utils/cassandra.js:85]: See my question on the Java implementation about whether we should continue to execute if SSL enabled but there is a problem with the cert file.

**Review by jeffreyscarpenter — COMMENTED** [review]
There are no changes to docker configuration in this PR, as there are in some of the equivalent PRs in other repos.

File: .gitignore (hunk 1 above)
Comment (SonicDMG) [reply .gitignore:24]: lol

Comment (Adron) [reply .gitignore:24]: lol  :1st_place_medal:

File: Dockerfile (hunk 2 above)
Comment (Adron) [reply Dockerfile:19]: Seems strange. I believe it had an ADD before vs. the COPY but it sure doesn't show it.  This worked last I tired it, so it seems ok regardless.

File: src/utils/cassandra.js (hunk 3 above)
Comment (Adron) [reply src/utils/cassandra.js:85]: @SonicDMG and I discussed this a bit, and if I recall agreed to go ahead and get this in as is, and work on figuring out how to determine if the cert file is ok via code at a later time. I wasn't sure exactly how to go about doing that at this point.

**Review by Adron — APPROVED** [review]
LGTM.

**Pull request merged**